dvc/repo/update.py

```diff
@@ -1,14 +1,20 @@
+from ..dvcfile import Dvcfile
 from . import locked
 
 
 @locked
-def update(self, target, rev=None):
-    from ..dvcfile import Dvcfile
+def update(self, targets=None, rev=None, recursive=False):
+    if not targets:
+        stages = self.collect(targets, recursive=recursive)
```

Comment:
We've explicitly avoided supporting `update` without targets to avoid possible issues with user trying to update multiple files with a revision that is not actually relevant for all of them (imagine 2 imports from 2 projects and `--rev v5` not existing in one of them). So let's not introduce that for now, let's keep `targets` a required argument.